packages/xo-server-sdn-controller/src/utils/tls-helper.js

```diff
@@ -0,0 +1,55 @@
+import fromEvent from 'promise-toolbox/fromEvent'
+import createLogger from '@xen-orchestra/log'
+import { connect } from 'tls'
+
+// =============================================================================
+
+const log = createLogger('xo:xo-server:sdn-controller:tls-connect')
+
+// =============================================================================
+
+export class TlsHelper {
+  updateCertificates(clientKey, clientCert, caCert) {
+    this._clientKey = clientKey
+    this._clientCert = clientCert
+    this._caCert = caCert
+    log.debug('Certificates have been updated')
```

Comment:
What do you mean by *updated*?

I don't see any certificate updates in this class.

Reply:
The 3 lines above the log changes the fields of the class with new certificates and keys.

Reply:
My bad, I thought it was a constructor :facepalm: